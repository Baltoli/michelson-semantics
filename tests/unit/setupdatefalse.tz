$Instructions { UPDATE } ; 
$Stack ListItem(2) ListItem(False) ListItem(SetItem(1) SetItem(3) SetItem(5) SetItem(7)) ;
