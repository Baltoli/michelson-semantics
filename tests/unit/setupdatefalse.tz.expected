<generatedTop>
  <k>
    .
  </k>
  <paramtype>
    #NotSet
  </paramtype>
  <storagetype>
    #NotSet
  </storagetype>
  <stack>
    Stack ( ListItem ( SetItem ( 1 )
    SetItem ( 3 )
    SetItem ( 5 )
    SetItem ( 7 ) ) )
  </stack>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( -1 )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( -2 )
  </sourceaddr>
  <senderaddr>
    #Address ( -3 )
  </senderaddr>
</generatedTop>
